lua/nvim-gps/init.lua

```diff
@@ -5,14 +5,44 @@ local gps_utils = require("nvim-gps.utils")
 
 local M = {}
 
+local default_separator = ' > '
+local default_icons = {
+	["class-name"] = ' ',
+	["function-name"] = ' ',
+	["method-name"] = ' ',
+	["container-name"] = '⛶ ',
+	["tag-name"] = '炙'
+}
+local withDefaultIcons = function(icons)
```

Comment:
Can this function handle current way of setting icons? or will this be a breaking change for users who want to set default icons?

Reply:
I think we can make it work, but I'd add a warn message for the current way of setup informing the users of the change. This way after a while we'd could keep only the new format.